ITKIOScanco_UnitConversion.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    print(metadata)
</pre><p>Can we save this cell's output in the notebook so it can be browsed on GitHub?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/ITKIOScanco/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I'm approaching this as saving the metadata to a .txt file. If you had something else in mind, please let me know.</p>